codex/stores/maintenance.nim

```diff
@@ -0,0 +1,96 @@
+## Nim-Codex
+## Copyright (c) 2023 Status Research & Development GmbH
+## Licensed under either of
+##  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+##  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+## at your option.
+## This file may not be copied, modified, or distributed except according to
+## those terms.
+
+## Store maintenance module
+## Looks for and removes expired blocks from blockstores.
+
+import pkg/chronos
+import pkg/chronicles
+import pkg/questionable
+import pkg/questionable/results
+
+import ./repostore
+import ../utils/timer
+import ../clock
+import ../systemclock
+
+const
+  DefaultBlockMaintenanceIntervalSeconds* = 10 * 60
```

Comment:
`std/times` also has human readable units, we should probably just use `10.minutes`.